docs/Common/Enums/index.md

```diff
@@ -664,6 +675,8 @@ Radio Buttons
 |LINE_IN|4||
 |IPOD|5||
 |MOBILE_APP|6||
+|CD|7||
```

Comment:
`CD` was added in the second spot in the XML, so it wouldn't have a value of `7`

Reply:
@jacobkeeler, fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/103/commits/4212ef5c22a5dfe946b9c4d0d8a1d63040d171eb